simplesat/package.py

```diff
@@ -85,15 +87,24 @@ def __init__(self, name, version, install_requires=None, conflicts=None):
                 (("MKL", ((">= 10.1", "< 11"),)),
                  ("nose", (("*",),)),
                  ("six", (("> 1.2", "<= 1.2.3"), (">= 1.2.5-2",)))
-
         conflicts : tuple(tuple(str, tuple(tuple(str))))
             A tuple of tuples mapping distribution names to disjunctions of
             conjunctions of version constraints.
 
             This works the same way as install_requires, but instead denotes
             packages that must *not* be installed with this package.
+        provides : iterable of package names
+            The packages that are provided by this distribution. Useful when
```

Comment:
This makes it sounds like only the names of actual packages can be specified. From the examples, it looks like this could be an arbitrary label listed as a requirement. Is this purely a semantic difference?